barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_integration.test.cpp

```diff
@@ -232,12 +232,10 @@ INSTANTIATE_TEST_SUITE_P(AcirTests,
                                          "brillig_pedersen",
                                          "brillig_recursion",
                                          "brillig_references",
-                                         //  "brillig_scalar_mul",
```

Comment:
all of these tests are not part of execution_success so not relevant for proving/verifying